nkms/policy/models.py

```diff
@@ -45,14 +46,14 @@ def __init__(self, alice, hrac, expiration, deposit=None, ursula=None,
         self.ursula = ursula
 
     def __bytes__(self):
-        return bytes(self.alice.seal) + bytes(
+        return bytes(self.alice.stamp) + bytes(
             self.hrac) + self.expiration.isoformat().encode() + bytes(
             self.deposit)
 
     @classmethod
     def from_bytes(cls, contract_as_bytes):
         contract_splitter = key_splitter + BytestringSplitter((bytes, KECCAK_DIGEST_LENGTH),
-            (bytes, 26))
+                                                              (bytes, 26))
```

Comment:
`26` should probably be a constant somewhere, imho.

Reply:
`TWENTY_SIX = 26` ?

I can do that.